src/openalea/deploy/command.py

```diff
@@ -449,8 +449,13 @@ def run(self):
                     print "Run SCONS install in conda environment"
 
                     if is_conda_build():
+                        # Fred remarks : Is it realy usefull ? Scons can be parameterized to find such repository
                         prefix = conda_prefix()
-                        externp += 'build_libdir=%s/lib build_bindir=%s/bin build_includedir=%s/include'%(prefix, prefix, prefix)
+                        if not ("posix" in os.name): prefix = os.path.join(prefix, 'Library')
```

Comment:
We can return the env variable set by CONDA. I would prefer another method in util.py like
def conda_library_prefix()